engines/instance_verification/config/environments/test.rb

```diff
@@ -6,5 +6,5 @@
 Rails.application.configure do
   config.cache_config_file = Rails.root.join('engines/registry/spec/data/rmt-cache-trim.sh')
   config.repo_cache_dir = 'repo/cache'
-  config.registry_cache_dir = 'registry/cache'
+  config.registry_cache_dir = 'test_registry/test_cache'
```

Comment:
Why does the test env need a different directory here, and not in the other cases ('repo_cache_dir')? If this is needed, I'd suggest to only use a different subdir, like 'registry/test_cache'. In general, the better place might have been 'tmp/cache/registry' but I guess that's too late to change?

Reply:
Then you can drop this change here, and tackle the cache directory updates in another pr.